docs/modules_overview.md

```diff
@@ -0,0 +1,228 @@
+---
+layout: index
+title: SW-Helpers
+navigation_weight: 0
+---
+
+<!-- DO NOT EDIT. This page is autogenerated. --><!-- To make changes, edit templates/README.hbs, not README.md! -->
```

Comment:
Is this comment still accurate? If so, shouldn't the changes be made to the version of this file in `templates/`?